spec/classes/puppet_metrics_collector_system_spec.rb

```diff
@@ -1,6 +1,10 @@
 require 'spec_helper'
 
 describe 'puppet_metrics_collector::system' do
+  # This define has an undeclared dependency on the main
+  # puppet_metrics_collector class.
+  let(:pre_condition) { 'include puppet_metrics_collector' }
```

Comment:
This is a behavior change. Does `puppet_metrics_collector` need to be included for the `system` class to work? If that is the case, we need to fix it.

Reply:
This was me copying/pasting while trying to get tests to work.  It likely does not need the dependency, so I'll try yanking this out.

Reply:
Should be good now.